DOCS/CBLAS.md

```diff
@@ -0,0 +1,143 @@
+# THE CBLAS C INTERFACE TO BLAS
+
+## Contents
+[1. Introduction](#1-introduction)
+
+[1.1 Naming Schemes](#11-naming-schemes)
+
+[1.2 Integers](#12-integers)
+
+[2. Function List](#2-function-list)
+
+[2.1 BLAS Level 1](#21-blas-level-1)
+
+[2.2 BLAS Level 2](#22-blas-level-2)
+
+[2.3 BLAS Level 3](#23-blas-level-3)
+
+[3. Examples](#3-examples)
+
+[3.1 Calling DGEMV](#31-calling-dgemv)
+
+[3.2 Calling DGEMV_64](#32-calling-dgemv_64)
+
+## 1. Introduction
+This document describes CBLAS, the C language interface to the Basic Linear Algebra Subprograms (BLAS).
+In comparison to BLAS Fortran interfaces CBLAS interfaces support both row-major and column-major matrix
+ordering with the `layout` parameter.
+The prototypes for CBLAS interfaces, associated macros and type definitions are contained in the header
+file [cblas.h](../CBLAS/include/cblas.h)
+
+### 1.1 Naming Schemes
+The naming scheme for the CBLAS interface is to take the Fortran BLAS routine name, make it lower case,
+and add the prefix `cblas_`. For example, the BLAS routine `DGEMM` becomes `cblas_dgemm`.
+
+CBLAS routines also support `_64` suffix that enables large data arrays support in the LP64 interface library
+(default build configuration). This suffix allows to mix LP64 and ILP64 programming models in one application.
```

Comment:
Grammatically, should be "allows mixing" instead of "allows to mix".

Reply:
Thank you for catching it! Fixed.